pandas/tests/tools/test_to_datetime.py

```diff
@@ -2116,7 +2116,7 @@ def test_dataframe_field_aliases_column_subset(self, df, cache, unit):
         result = to_datetime(df[list(unit.keys())].rename(columns=unit), cache=cache)
         expected = Series(
             [Timestamp("20150204 06:58:10"), Timestamp("20160305 07:59:11")],
-            dtype="M8[ns]",
+            dtype="M8[us]",
```

Comment:
The to_datetime behaviour also changed because of this PR?

Reply:
Yes.  to_datetime on a DataFrame calls to_timedelta.